phidgets_ik/src/ik_ros_i.cpp

```diff
@@ -49,6 +49,7 @@ void IKRosI::initDevice()
     snprintf(topicname, sizeof(topicname), "DigitalOutput%02d", i);
     boost::shared_ptr<OutputSetter> s (new OutputSetter(ik_handle_, i));
     s->subscription = nh_.subscribe(topicname, 1, &OutputSetter::set_msg_callback, s);
+    s->service = nh_.advertiseService(topicname, &OutputSetter::set_srv_callback, s);
```

Comment:
The services should be called "set_digital_output00", not "DigitalOutput00". Also, (although it's a bit unrelated to this PR, but it should be changed nonetheless), the other topics should be called "digital_output00", "digital_input00", analog_input00".